src/asyncMiddleware/asyncMiddleware.ts

```diff
@@ -12,12 +12,16 @@ import { Middleware } from 'koa';
  */
```

Comment:
We should document the parameter and what happens if it's unset in this TSDoc. Can we also update the README example with a TTL?